app/frontend/src/features/communities/CommunityPage/DiscussionsSection.tsx

```diff
@@ -0,0 +1,153 @@
+import TextBody from "../../../components/TextBody";
+import { Community } from "../../../pb/communities_pb";
+import { routeToCommunityDiscussions } from "../../../routes";
+import { useListDiscussions, useNewDiscussionMutation } from "../useCommunity";
+import { useCommunityPageStyles } from "./CommunityPage";
+import DiscussionCard from "./DiscussionCard";
+import SectionTitle from "./SectionTitle";
+
+const useStyles = makeStyles((theme) => ({
+  discussionsHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  newPostButton: {
+    margin: theme.spacing(1),
+  },
+  discussionsContainer: {
+    display: "flex",
+    flexDirection: "row",
+    flexWrap: "wrap",
+    justifyContent: "space-between",
+    "& > *": {
+      width: "100%",
+      [theme.breakpoints.up("sm")]: {
+        width: `calc(50% - ${theme.spacing(1)})`,
+      },
+      [theme.breakpoints.up("md")]: {
+        width: `calc(33.33% - ${theme.spacing(1)})`,
+      },
+    },
+    //preserve grid in the last row
+    "&::after": {
+      content: "''",
+      flexBasis: "100%",
+      [theme.breakpoints.up("sm")]: {
+        flexBasis: `calc(50% - ${theme.spacing(1)})`,
+      },
+      [theme.breakpoints.up("md")]: {
+        flexBasis: `calc(33.33% - ${theme.spacing(1)})`,
+      },
+    },
+  },
+  discussionCard: {
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export default function DiscussionsSection({
+  community,
+}: {
+  community: Community.AsObject;
+}) {
+  const classes = { ...useCommunityPageStyles(), ...useStyles() };
+
+  //temporary
+  const [isNewCommentOpen, setIsNewCommentOpen] = useState(false);
+
+  const {
+    isLoading: isDiscussionsLoading,
+    error: discussionsError,
+    data: discussions,
+    hasNextPage: discussionsHasNextPage,
+  } = useListDiscussions(community.communityId);
+
+  const queryClient = useQueryClient();
+  const newDiscussionMutation = useNewDiscussionMutation(queryClient);
+
+  return (
+    <>
+      <div className={classes.discussionsHeader}>
+        <SectionTitle icon={<EmailIcon />}>
+          {`${community.name} discussions`}
+        </SectionTitle>
+        <IconButton
+          aria-label="New post"
+          onClick={() => setIsNewCommentOpen(true)}
+        >
+          <AddIcon />
+        </IconButton>
+      </div>
+      {discussionsError && (
+        <Alert severity="error">{discussionsError.message}</Alert>
+      )}
+      {
+        //This comment adding dialog is temporary
+      }
+      <Dialog
+        open={isNewCommentOpen}
+        onClose={() => setIsNewCommentOpen(false)}
+      >
+        <DialogContent>
+          {newDiscussionMutation.error && (
+            <Alert severity="error">
+              {newDiscussionMutation.error.message}
+            </Alert>
+          )}
+          <NewComment
+            onComment={async (content) =>
+              newDiscussionMutation.mutate({
+                title: "test",
```

Comment:
This is just placeholder to see if the discussions actually work

Reply:
But if the API is already there, shouldn't we just let the title be set? Or is it still subject to change?

Reply:
I didn't want to put any more in this PR - this dialog is just the minimum for me to see if the discussions actually appeared since there was no mock data